apps/console/src/app/pages/projects/getting-started-wizard/TypeScriptOpenAIIntegrationTutorial.tsx

```diff
@@ -0,0 +1,127 @@
+import { Button, Typography } from "antd";
+import { Highlight, themes } from "prism-react-renderer";
+import { useCurrentProject } from "../../../lib/hooks/useCurrentProject";
+import styled from "@emotion/styled";
+import {
+  Usage,
+  useGettingStartedWizard,
+} from "../../../lib/providers/GettingStartedWizardProvider";
+
+const StyledPre = styled.pre`
+  background: #000;
+  border-radius: 8px;
+  padding: 14px;
+`;
+
+const HighlightCode = ({ code }) => (
```

Comment:
Maybe it's time to make it a reusable component (you don't need to do anything, just put it in a different file inside the components folder and export it)